### PR TITLE
fix more in anvil

### PR DESCRIPTION
**Pull request by NycuRO**
### Description

Fix more in anvil

### Reason to modify

Anvil have TODO. Need fix that.
### Tests & Reviews

Tested and work fine.

### CODE REVIEW

File: src/pocketmine/block/Anvil.php [hunk 1]
```diff
@@ -77,21 +65,23 @@ public function onActivate(Item $item, Player $player = null){
...
+			$this->meta = ($this->meta & 0x0c) | $direction;
+			$this->getLevel()->setBlock($block, $this, true, true);
+			return true;
+		}
+		return false;
 	}
-
-	public function getDrops(Item $item) : array {
+	public function getDrops(Item $item) : array{
+		$damage = $this->getDamage();
```
Comment (dktapps) [inline src/pocketmine/block/Anvil.php]: This is completely pointless.

**Comment by dktapps** [comment]
Formatting is a mess and you haven't changed anything that needs to be changed. AND you used GitHub Web Editor. Didn't you read the contribution guidelines? -1.

**Comment by dktapps** [comment]
Not to mention the fact you copied wrong code from ClearSky.

**Comment by NycuRO** [comment]
@dktapps what is wrong ,lmao?

**Pull request closed without merging**